### PR TITLE
Transponder: add tslc in AdsbVehicle

### PR DESCRIPTION
**Pull request by chep**
related to https://github.com/mavlink/MAVSDK/pull/1730

### CODE REVIEW

**Review by JonasVautherin — APPROVED** [review]

**Review by JonasVautherin — CHANGES_REQUESTED** [review]
> protos/transponder/transponder.proto:68:9:"uint8_t" is not defined.

Actually it seems like uint8_t is not supported in protobuf.

File: protos/transponder/transponder.proto [hunk 1]
```diff
@@ -65,6 +65,7 @@ message AdsbVehicle {
     string callsign = 9 ; // The callsign
     AdsbEmitterType emitter_type = 10; // ADSB emitter type.
     uint32 squawk = 13; // Squawk code.
+	uint8_t tslc = 14; // Time Since Last Communication.
```
Comment (JonasVautherin) [inline protos/transponder/transponder.proto:68]:
```suggestion
    uint8_t tslc = 14; // Time since last communication.
```

Comment (chep) [reply protos/transponder/transponder.proto]: fixed

File: protos/transponder/transponder.proto [hunk 2]
```diff
@@ -65,6 +65,7 @@ message AdsbVehicle {
     string callsign = 9 ; // The callsign
     AdsbEmitterType emitter_type = 10; // ADSB emitter type.
     uint32 squawk = 13; // Squawk code.
+    uint32 tslc = 14; // Time Since Last Communication.
```
Comment (julianoes) [inline protos/transponder/transponder.proto:68]: What's the unit?

**Pull request merged**